tests/cli/check/--deny-warnings/warning_becomes_error_with_flag/inputs/unused-input.wdl

```diff
@@ -0,0 +1,10 @@
+## Unused input.
```

Comment:
For each of these, can you just add a bit more context that is specific to each test in the doc comment at the start of the WDL files?

Reply:
I believe I've updated all the wdl files to have appropriate comments describing the tests